src/lib_json/json_writer.cpp

```diff
@@ -1168,9 +1168,7 @@ StreamWriter* StreamWriterBuilder::newStreamWriter() const {
   const bool emitUTF8 = settings_["emitUTF8"].asBool();
   unsigned int pre = settings_["precision"].asUInt();
   CommentStyle::Enum cs = CommentStyle::All;
-  if (cs_str == "All") {
-    cs = CommentStyle::All;
-  } else if (cs_str == "None") {
+  if (cs_str == "None") {
```

Comment:
This seems wrong. Am I missing something here?

Reply:
https://github.com/open-source-parsers/jsoncpp/blob/ceae0e3867fe16e1227b4a39fe6951ee005591dc/src/lib_json/json_writer.cpp#L1170